### DISCUSSION TITLE
Edit the welcome page

### DISCUSSION THREAD

**Discussion by ptr1120** [keycloakify/keycloakify#97] [Q&A] [answered]
Hello,

is it possible to edit the welcome page with `keycloakify`?

**Comment by garronej** [comment, marked as answer]
Hi @ptr1120,  
I assume you are referring to this page:  
<img width="1179" alt="image" src="https://user-images.githubusercontent.com/6702424/166156792-bc26be38-8cd5-4377-b1e6-da5e8fae5839.png">

Unfortunately, no, not yet at least.  
<del>This page belong to the `account` set of pages.  </del> no it's it's own theme named "wellcome"

https://docs.keycloakify.dev/limitations#login-and-email-only

<img width="1544" alt="image" src="https://user-images.githubusercontent.com/6702424/166156926-f03e8981-e595-485b-b1d2-eb00063322fc.png">

**Reply by ptr1120** [reply]
thank you @garronej  this was the page I meant.

**Comment by garronej** [comment]
It's now planed, there's an issue about it:  https://github.com/keycloakify/keycloakify/issues/148
